src/smart-components/group/add-group/add-group-wizard.js

```diff
@@ -72,7 +72,8 @@ const AddGroupWizard = ({
   return (
     <Wizard
       isLarge
-      title={ 'Add group' }
+      title={ 'Create and configure a group' }
```

Comment:
No need to wrap in object, you can directly write `title="Create and configure a group"` same for description

Reply:
Oh, thanks! Totally missed it, just followed the code pattern